tests/testthat/test-cv.R

```diff
@@ -1,6 +1,19 @@
+test_that("cv passes correct parameters to inner solvers (when lamdba is not
+          specified)", {
+            y <- c(1, rpois(100, dnorm(1:100, 50, 15) * 500 + 1))
+            cv <- cv_estimate_rt(y, korder = 3, nfold = 3, nsol = 30)
+            mod <- estimate_rt(y, korder = 3, nsol = 30)
 
+            expect_identical(cv$lambda, mod$lambda)
+          })
 
-test_that("fold index helper function works", {
-  skip("This function was removed.")
-
+test_that("test CV returns an error message when max iteration is too low for
```

Comment:
If you're going to do this, set the `maxit` to be small (rather than `nsol` to be large). And adjust in light of the above change to warning.